Binary_search/Binary_search.py

```diff
@@ -0,0 +1,39 @@
+# Binary search task A_2
+# https://informatics.msk.ru/mod/statements/view.php?id=192
+
+'''
+n, k = [int(i) for i in input().split()]
+
+N = [int(i) for i in input().split()]
+K = [int(i) for i in input().split()]

```

Comment:
Another naming suggestion
```suggestion
numbers_to_find = [int(number) for i in input().split()]
```